salt/metalk8s/repo/online.sls

```diff
@@ -1,6 +1,6 @@
 {%- from "metalk8s/map.jinja" import repo with context %}
 
-Configure EPEL repository:
+Configure epel repository:
   pkg.installed:
   {%- if grains["os"] == "CentOs" %}
```

Comment:
not related to your code but is this a typo ? `CentOS`

Reply:
It is :)

Reply:
fixed